lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -2621,7 +2622,7 @@ sub observation_units_POST {
 	my $self = shift;
 	my $c = shift;
 	# The observation units need an operator, so login required
-	my $force_authenticate = 1;
+	my $force_authenticate = 0;
```

Comment:
Can these values be driven by the `brapi_require_login` config in `sgn_local.conf`?

Reply:
The `brapi_require_login` is only used to determine if brapi login is required for all of the endpoints. The force authenticatation ability was added to force authentication on endpoints where logging in isn't optional because the user information is needed for the process. The `_authenticate_user` method in `repos/sgn/lib/SGN/Controller/AJAX/BrAPI.pm` has the logic for this.

Reply:
I think we can refactor this to make it cleaner in this card: https://breedinginsight.atlassian.net/browse/BI-1037. It's not very clean as it is now.